pep_sphinx_extensions/pep_theme/static/style.css

```diff
@@ -1,17 +1,49 @@
 @charset "UTF-8";
 /* Styles for PEPs */
 
-/* Set master 'light-mode' colours (default) */
+/*
+ * `initial` works like undefined variables, so `var(intial, x)` will resolve to `x`.
+ * A space means an empty value, so `var( , x) y` will resolve to `y`.
+ */
+@media (prefers-color-scheme: dark) {
+   :root {
+    --light: ;
+    --dark: initial;
+  }
+}
+
+@media (prefers-color-scheme: light) {
+   :root {
+    --dark: ;
+    --light: initial;
+  }
+}
+
+:root[data-colour_scheme="dark"] {
+  --light: ;
+  --dark: initial;
+}
+
+:root[data-colour_scheme="light"] {
+  --dark: ;
+  --light: initial;
+}
+
+/* Set master colours */
 :root {
-    --colour-background: white;
-    --colour-background-accent: #eee;
-    --colour-text: #333;
-    --colour-links: #0072aa;
-    --colour-scrollbar: #ccc;
-    --colour-rule-strong: #888;
-    --colour-rule-light: #ddd;
-    --colour-inline-code: #f8f8f8;
-    --colour-warning: #fee;
+    --colour-background: var(--light, white) var(--dark, #011);
```

Comment:
This is cool, I didn't know you could inline the flags like this!

Reply:
I’m also happy I discovered this! I think it’s much better than having two blocks or even two files.